### PR TITLE
Error handling for Android

### PR DESCRIPTION
**Pull request by tilak-puli**
Add Exception handler for BLE module, wallet and verifier to handle exceptions.
Catch exception and pass them to handlers in statehandler and transfer handler to avoid handlers quitting.
Re-Throw GCM Exception to exception handlers

### CODE REVIEW

File: android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt [hunk 1]
```diff
@@ -61,7 +60,7 @@ class Peripheral(context: Context, peripheralListener: IPeripheralListener) {
 
   fun sendData(serviceUUID: UUID, charUUID: UUID, data: ByteArray) {
     val currentState = messageSender.getCurrentState()
-    if (currentState == StateHandler.States.CommunicationReady) {
+    if (currentState == StateHandler.States.CommunicationReady || (serviceUUID === GattService.DISCONNECT_CHAR_UUID)) {
```
Comment (krishnakumar4a4) [inline android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt]: Why is this needed? 
Also the condition should be `charUUID === GattService.DISCONNECT_CHAR_UUID` ?

Comment (tilak-puli) [reply android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt]: Yes. it has to be charUUID. I will update this. 

In a case, where communication not ready yet and state is connected. And if disconnect happens, Verifier is not able to send disconnect notification. I don't remember facing this in current state. but when I tried throwing exception onConnected. this was happening.

Comment (tilak-puli) [reply android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt]: Fixed this in a new commit

Comment (tilak-puli) [reply android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt]: Fixed this in a new commit

**Comment by krishnakumar4a4** [comment]
Following new exception cases are included:
1. onMTUChanged callback if MTU size is less than `MIN_MTU_REQUIRED = 64`
2. Percentable of missed chunks is more than 70% on Transfer report request.
